Sources/Components/ReviewButtonView/ReviewButtonView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public protocol ReviewButtonViewDelegate: AnyObject {
+    func reviewButtonView(_ reviewButtonView: ReviewButtonView, giveReviewWasTapped startReview: Bool)
+}
+
+public final class ReviewButtonView: UIView {
+    // MARK: - Public
+
+    public weak var delegate: ReviewButtonViewDelegate?
+    public var text: String = "" {
+        didSet {
+            reviewButtonControl.text = text
+        }
+    }
+
+    // MARK: - Private
+
+    private let reviewButtonControlHeight: CGFloat = 48
+    private let viewHeight: CGFloat = 64
```

Comment:
Wouldn't just `height` read better(?) since I guess `self` is the view in question. (Makes the constraint setup in `addToView` more readable, I believe.